### ISSUE TITLE
Previewing pages when Manager is split from front end

### ISSUE THREAD

**Issue by monsieurberry** [PiranhaCMS/piranha.core#1910]
Hi,

We recently split out our CMS Manager out to a seperate App Service which literally just contains the Manager packages and config.

What we quickly found is that the preview links no longer work. Most probably because the required Controllers / Css etc are not available on this Manager App.

I had a look through previously logged issues and the cloest I could find is this https://github.com/PiranhaCMS/piranha.core/issues/335

Is there a way I can get the previewing to work somehow, despite the relevant code being remote?

Cheers

**Comment by monsieurberry** [comment]
Ah, it appears I didnt look hard enough - I see https://github.com/PiranhaCMS/piranha.core/issues/1291 detailing a similar issue.

**Comment by tidyui** [comment]
There are more issues with solving this. For example user authentication across multiple applications.

**Comment by monsieurberry** [comment]
Yes, never thought of that.

I got around it (for now) by creating a simple Controller in my Manager app with a wildcard action for Page/Archive/Post.

I then copied the DisplayTemplates from my main app (these hardly ever change), and add the Internal Site Id of the site to ViewData in order to set the <base href /> to my live site to grab the css/js (required setting up a CORS rule in main app).

It's grossly inelegant, and doesn't give me the full Preview experience, but its close enough, and allows my content editors to see the content before they publish.

Thanks for all your efforts @tidyui
